Client/gui/CGUI_Impl.cpp

```diff
@@ -165,9 +174,8 @@ void CGUI_Impl::SetSkin(const char* szName)
     // Destroy any windows we already have
```

Comment:
`m_pTop` UAF + race condition.. line 174-177 are executing it while invalid. Becomes valid again at line 178: Cleanup()